### PR TITLE
commit message

### PR DESCRIPTION
**Pull request by Richmond-Andoh**

### CODE REVIEW

File: CorsaSchoolManagementSystem/seeder.php [hunk 1]
```diff
@@ -4,9 +4,9 @@
 $db = new Database();
```
Comment (OliverMensahDev) [inline CorsaSchoolManagementSystem/seeder.php:4]: What's the point of changing this?

File: CorsaSchoolManagementSystem/Views/dashboard.php [hunk 2]
```diff
@@ -180,7 +180,7 @@
                             <?php foreach($studentDetails as $details): ?>
                             <tr>
                                 <td><img src="<?php echo BASE_URL . "/" .$details->picture?>" width="50" height="50" class="img-thumbnail" alt="<?php echo $details->firstname ."'s profile picture" ?>"/></td>
-                                <td><?php echo $details->firstname . ' ' . $details->middlename. ' ' .$details->lastname ?></td>
+                                <td><a href="<?php echo BASE_URL ?>/Views/studentDetails.php" class="text-decoration-none text-dark"><?php echo $details->firstname . ' ' . $details->middlename. ' ' .$details->lastname ?></a></td>
```
Comment (OliverMensahDev) [inline CorsaSchoolManagementSystem/Views/dashboard.php:183]: You don't use views for application URLs, use controllers for this

**Review by OliverMensahDev — COMMENTED** [review]
Also, the images are not needed here. Using them to test on your device doesn't mean they are needed for the general application

File: CorsaSchoolManagementSystem/Views/dashboard.php (hunk 2 above)
Comment (Richmond-Andoh) [reply CorsaSchoolManagementSystem/Views/dashboard.php:183]: That was Sammy’s branch but after his commit to the branch he couldn’t find the pull request on his machine but I could find it on mine so I did the pr for him

Comment (OliverMensahDev) [reply CorsaSchoolManagementSystem/Views/dashboard.php:183]: Okay! You can make the changes for him or just inform to do those changes

Comment (Richmond-Andoh) [reply CorsaSchoolManagementSystem/Views/dashboard.php:183]: Alright I wiil do that

**Pull request closed without merging**